### PR TITLE
#85: Rename Atlas to ZalandoCommerce (epic)

### PR DESCRIPTION
**Pull request by bonkey**
#85: Rename Atlas to ZalandoCommerce

### CODE REVIEW

File: Sources/ZalandoCommerceAPI/ZalandoCommerceAPI/Utils/Keychain.swift [hunk 1]
```diff
@@ -86,7 +86,6 @@ struct Keychain {
         }
         let entries: [[AnyHashable: Any]] = results.flatMap { $0 as? [AnyHashable: Any] }
 
-        Logger.error(entries)
```
Comment (hani-ibrahim) [inline Sources/ZalandoCommerceAPI/ZalandoCommerceAPI/Utils/Keychain.swift]: 😂

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Pull request merged**